src/components/ExperimentDetail.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react'
+// import { Link } from 'react-router-dom'
+import { Container, Row, Col, Alert, Card, Progress, CardBody, CardTitle, CardSubtitle, CardText, FormGroup, Label, Input, CustomInput, ButtonDropdown, DropdownToggle, DropdownMenu, DropdownItem } from "reactstrap";
+import "../static/css/Experiments.css";
+ 
+
+const ExperimentDetail = () => {
+
+    const [data] = useState([
+        { id: "1", emojis: "🚀💸", name:"Up Only WBTC" , contract: "OxSalo12345", assetPrice: "1.00", currency: "$", totalAssets: "1000", totalAUM: "10000", baseAsset: "WBTC"},
+      ]);
+
+    const [dropdownOpen, setOpen] = useState(false);
+
+    const toggle = () => setOpen(!dropdownOpen);
+
+    return (
+        <Container className="experiments-container container h-100">
+            <Row className="row h-100 justify-content-center align-items-center">
+                <Col className="experiments-column col-12">
+                <div>
+                    <Card className="card card-detail">
+                    <div>
+                        {data.map(item => (
```

Comment:
Is this going to be a dropdown that lets users switch between the different experimental products?

Reply:
Nop, didn't imagine it that way